wifiphisher/common/phishinghttp.py

```diff
@@ -20,6 +20,24 @@ def get(self):
         self.redirect("http://10.0.0.1:8080/")
 
 
+class ValidateHandler(tornado.web.RequestHandler):
+
+    def initialize(self, em):
+        self.em = em
+
+    def get(self, cred_type, cred_content):
+        """
+        Override the get method
+
+        :param self: A tornado.web.RequestHandler object
+        :type self: tornado.web.RequestHandler
+        :return: None
+        :rtype: None
+        """
+        value = self.em.verify_cred((cred_type, cred_content))
+        self.write("%s" % value)
+
```

Comment:
From within the phishing scenario, you should call `/validate/psk/[value_of_psk]` using AJAX. Based on the result (`True` or `False`) you may print back to the victim user whatever you want.